### DISCUSSION TITLE
如何获取非实体对象的Id

### DISCUSSION THREAD

**Discussion by Leovany** [kbengine/kbengine#1501] [General]
account 是实体，其他模块是挂在account 实体上
那请问下，改如何获得 这个Id ( 对应 数据库唯一ID ）

![1111111111](https://user-images.githubusercontent.com/2864116/132640992-1185d9ba-fce7-476f-8775-7040051ecd5b.png)

**Comment by kbengine** [comment]
这是基于面向对象的引擎，目前没有暴露到这一层。 你只能获得实体的databaseID



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年9月9日(星期四) 下午3:23
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [kbengine/kbengine] 如何获取非实体对象的Id (#1501)





 
account 是实体，其他模块是挂在account 实体上
 那请问下，改如何获得 这个Id ( 对应 数据库唯一ID ）
 

 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by Leovany** [reply]
好的，感谢柯大
